src/prop-types.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types'
+
+export const FavoriteDaoType = PropTypes.shape({
+  name: PropTypes.string,
+  address: PropTypes.string,
+  favorited: PropTypes.bool,
+})
+
+export const DaoItemType = PropTypes.shape({
+  name: PropTypes.string,
+  address: PropTypes.string,
+})
+
+export const DaoAddressType = PropTypes.shape({
+  address: PropTypes.string,
```

Comment:
It can be empty when we are on the onboarding, or before the address gets validated. Another option could be to pass e.g. `null` instead of the object until we have an address, but I don’t think having an empty address object is a big issue.